src/components/authPageComponents/AuthCardInputField.jsx

```diff
@@ -0,0 +1,24 @@
+import { TextField } from '@mui/material';
+import { authCardInput } from "../../utils/commonStyles";
+import { useAuth } from "../../contexts/";
+
+const AuthCardInputField = ({ inputProps }) => {
+
+    const { handleChange, errorAttr } = useAuth();
+
+    return (
+        <TextField
+            error={errorAttr(inputProps.uniqueId, inputProps.text)}
+            onChange={(e) => handleChange(e, inputProps.uniqueId)}
+            sx={authCardInput}
+            helperText={inputProps.text}
+            id={inputProps.uniqueId}
+            label={inputProps.inputLabel}
+            type={inputProps.inputType}
```

Comment:
If possible you can destructure input props, then you don't have to write inputProps. every time

Reply:
yeah, thanks for pointing that out, will work on your suggestion!!
thankyou so much for reviewing!!